src/url-params/url-params.js

```diff
@@ -188,14 +233,14 @@
                 urlManager.updateUrl();
 
                 // Get tags that need to be activated, based on current URL
-                var tagsToShow = urlManager.getTagsForCurrentState();
+                var hotspotTagsToClick = urlManager.getTagsForCurrentState();
```

Comment:
At this point they are actually tags applied in the Studio via the SDK Pallet. I'd sooner keep it called "tags" as that seems more consistent with the Studio's terminology. Make sense?